src/ui/Typeahead.tsx

```diff
@@ -128,7 +133,9 @@ const Typeahead: TypeaheadFunc = forwardRef(
         selected={selected}
         inputProps={{
           id,
+          type: inputType ?? 'text',
```

Comment:
```suggestion
          type: inputType,
```

This can go since you use defaultProps